### PR TITLE
Proposal quotation delivery

### PR DESCRIPTION
**Pull request by TheMasterProgrammer**
Delivery Request features to be merged with the main branch

### CODE REVIEW

**Review by ImenKhezzar — CHANGES_REQUESTED** [review]
You need to pull main into your branch

File: packages/client/src/routes/delivery_request.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+      console.error('Error creating package:', error);
+      setError("Failed to create package in the database.");
+    }
+  };
+
+  const shippingMethods = getShippingMethods();
+
+  return (
+    <div className="p-4 max-w-md mx-auto bg-white shadow-lg rounded-lg">
+      <Navbar />
```
Comment (Crystalz03) [inline packages/client/src/routes/delivery_request.tsx:79]: You should encapsulate the entire return statement with <> and </> and put the navbar outside the <div className...>

File: packages/client/src/routes/delivery_request.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+        </div>
+      )}
+
+      {cost !== null && (
+        <div className="mt-4 p-3 bg-green-100 text-green-800 font-semibold rounded">
+          Estimated Shipping Cost: ${cost.toFixed(2)}
+        </div>
+      )}
+      <Footer />
+    </div>
```
Comment (Crystalz03) [inline packages/client/src/routes/delivery_request.tsx:207]: Same thing at the navbar put the footer outside the div and inside the </>

File: packages/client/src/main.tsx [hunk 3]
```diff
@@ -37,6 +43,17 @@ const router = createBrowserRouter([
...
+      {
+        path: "test",
+        element: <Test />,
+        loader: testLoader,
+      },
+      {
+        path: "quotation",
+        element: <Quotation />,
+      },
+    ],
```
Comment (Crystalz03) [inline packages/client/src/main.tsx:56]: I don't think these should be children elements of the payment page. They should be on the same level as the other pages.

**Review by Crystalz03 — CHANGES_REQUESTED** [review]

**Review by ImenKhezzar — APPROVED** [review]

**Review by Crystalz03 — APPROVED** [review]

**Pull request merged**